package.exs

```diff
@@ -1,19 +0,0 @@
-defmodule Poolgirl.Mixfile do
```

Comment:
I'm not sure this file is needed.

Reply:
This file is needed in order to publish to Hex.

Alternatively, there's `rebar3_hex` plugin (which is likely a better option.)